modules/core/output_modules.php

```diff
@@ -1188,22 +1189,23 @@ class Hm_Output_timezone_setting extends Hm_Output_Module {
      */
     protected function output() {
         $zones = timezone_identifiers_list();
-        $settings = $this->get('user_settings', array());
+        $settings = $this->get('user_settings', array());        
         $reset = '';
+        // exit(var_dump($settings));
         if (array_key_exists('timezone', $settings)) {
             $myzone = $settings['timezone'];
         }
         else {
-            $myzone = false;
+            $myzone = array_key_exists('default_setting_timezone', config('app')) ? config('app')['default_setting_timezone'] : 'UTC';
```

Comment:
Why wouldn't config app be defined? I don't think you should use config function but simply $this->config - it should be set in modules.